doc/url_parameters.md

```diff
@@ -1,42 +1,45 @@
+| `t` (`t1` for B state) | date | *`YYYY-MM-DD-Thh:mm:ssZ`* | Selected UTC day and time.\*\* |
+| `z` | number | <b>`1`</b> to <b>`5`</b> | The timescale axis zoom value from 1 to 5.\* |
+| `i` | number | <b>`1`</b> to <b>`5`</b> | The timeline interval value from 1 to 5.\* |
+| `ics` | boolean | <i>`true`</i> or <i>`false`</i> | If `custom selected` is `true`, a `custom interval` and `custom delta` will be active and allow custom intervals in time changes. |
+| `ici` | number | <b>`1`</b> to <b>`5`</b> | The `custom interval` value from 1 to 5\* identifies which time unit is changed by the date change arrows and optionally animation. *\*Only active and saved in url if `custom selected` is true.* |
+| `icd` | number | <b>`1`</b> to <b>`999`</b> | The `custom delta` value for how many `custom interval` time units are added or subtracted when using the date change arrows and optionally animation. *\*Only active and saved in url if `custom selected` is true.* |
+| `e` | string,date | *`event_id,yyyy-mm-dd`* | If any value is present, the events tab will be activated. If the value is a valid event\_id, the corresponding event will be highlighted in the event list. If a date (YYYY-MM-DD) is added to the event\_id, then the selected event for the specified date will be shown. |
+| `efs` | boolean | <i>`true`</i> or <i>`false`</i> | Determines whether to "show all" events that match the date range and category filters or, to also include the current map viewport bounding box in the event API call to limit results to that area.<br><br>Default: `true` (which means show all; no bbox included) |
+| `efd` | string | `yyyy-mm-dd,yyyy-mm-dd` | The event start and end dates to be included in an event API request. Default range is last 120 days from current app load time. |
+| `efc` | string | `id,id,id` | The event category ids for each category to be included in an event API request. |
+| `s` | string | *`coordinates`* | A pair of coordinates using Decimal Degrees format (`DDD.DDDD,DDD.DDDD`) to add a Location Search marker onto the map. *\*Only one marker pair of coordinates can be included in the permalink.* |
+| `ab` | boolean | **`on`** | If set to "on", the animation widget will be shown. |
+| `as` | date | *`YYYY-MM-DDThh:mm:ssZ`* | The animation start day & time.\*\* |
+| `ae` | date | *`YYYY-MM-DDThh:mm:ssZ`* | The animation end day & time.\*\* |
+| `av` | number | <b>`1`</b> to <b>`10`</b> | The animation speed value from 1 to 10. 1 = slowest, 10 = fastest. |
+| `al` | boolean | <i>`true`</i> or <i>`false`</i> | If any value is set, the animation loop will be turned on. Animation looping is disabled by default. |
+| `ca` | boolean | <i>`true`</i> or <i>`false`</i> | Determines if the A or B state is active. If this parameter exists at all, compare mode will be active. If `ca=true`, Compare mode will be active in the A state. |
+| `cm` | string | <i>`swipe`</i> , <i>`spy`</i> or <i>`opacity`</i> | If comparison mode is active (\`ca=true |
```

Comment:
The `cm`/ `cv` changes are missing.

Reply:
Good catch, I tried that WYSIWYG markdown editor plugin and it reformatted the whole file, I wonder if that caused this somehow? 🤔  Either way; fixed.